client/js/components/CardEditorComponent.js

```diff
@@ -8,11 +8,14 @@ if (window.FC === undefined) { window.FC = {}; }
       evt.preventDefault();
 
       var cb = () => {
-        ReactRouter.browserHistory.goBack();
+        // ReactRouter.browserHistory.goBack();
```

Comment:
If we add this alert won't it be the same number of clicks as before defeating the purpose of user-ability?